backend/scrapers/police_coders_groups/recompiler.py

```diff
@@ -5,6 +5,18 @@
 from data.community_measures.QC.QC import QCFilter
 
 
+# Initialise variables
+groups = []
+welshGroups = 0
+nonWelshGroups = 0
+exceptions = 0
+exceptions_names = []
+display0 = 0
+pointsInLAs = 0
+geomWelshGrps = []
+duplicates = 0
```

Comment:
Why these globals? 
You either don't need this initialisation at all - or otherwise you're doing something wrong here. 
Would not worry too much now as this module is way doing too much - also I don't get why this is called `recompiler`? 

The big caveats here is that `compilation` is a precise technical term to me referring specifically to something related to languages.. that's probably why I don't get the intended meaning :)